test/Table_Tests/src/Database/Postgres_Spec.enso

```diff
@@ -208,7 +208,7 @@ postgres_specific_spec suite_builder create_connection_fn db_name setup =
                 ["EUC_JIS_2004", "LATIN6", "LATIN8", "MULE_INTERNAL", "SHIFT_JIS_2004"]
 
             known_encodings.each encoding_name->
-                encoding = Postgres_Entity_Naming_Properties.parse_postgres_encoding encoding_name
+                encoding = parse_postgres_encoding Nothing encoding_name
```

Comment:
This test is showing a SRP problem with `parse_postgres_encoding` - the method tries to do too much.

I think we should split it into 2 parts:
1. simple 'functional' method `Text -> Encoding` that gets an encoding name and tries to interpret it and return an `Encoding` instance. Such a method can be easily unit tested.
2. a method that fetches the encoding from `JDBC_Connection` using `get_pragma_value` and then tries to parse it using (1).